index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mohammed Salah</title>
+
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <div class="header__brand-container">
+                <img src="./images/logo.png" alt="logo" />
+            </div>
+            <ul class="header__nav-container">
+                <li class="header__nav-container__nav-item"><a href="#">Portoflio</a></li>
+                <li class="header__nav-container__nav-item"><a href="#">About</a></li>
+                <li class="header__nav-container__nav-item"><a href="#">Contact</a></li>
+            </ul>
+            <div class="header__nav-container__mobile_menu">
+                <img src="./images/menu.svg" alt="hamburgar menu" />
+            </div>
+        </header>
+        <div class="intro__container ">
```

Comment:
Kindly use the` header` tag instead of the `div` tag, this is more semantic.

Reply:
Dear @keroloussamy,
I finished the changes as you mentioned in the previous comment but I close the pull request and opened a new one (I was thinking that this is the right way to do it) but I can't see where to request a new code review in the dashboard so can you please check the new pull request for the changes

 https://github.com/Mo7amedSMahdi/Personal-Portfolio/pull/5

With Appreciation.